controllers/cloud.redhat.com/namespacereservation_controller.go

```diff
@@ -230,18 +243,15 @@ func (r *NamespaceReservationReconciler) reserveNamespace(ctx context.Context, r
 		return err
 	}
 
+	defer func() {
```

Comment:
Only just thought of this, but I question the use of putting this at the end. If the function crashes out earlier, we won't assign the defer. If we want to calculate it only at the end on a successful run, why defer.....

Reply:
This is also just a count, so, I think we should be clear what and why we are counting, total? or total succeeded?

Reply:
I took out the defer and specified that it will count only successful reservations. By doing this, I left it at the end. If you prefer the total, including failed ones, I can update this. :smile: